code/signs-markings/signs-markings.css

```diff
@@ -36,7 +36,7 @@ div.view_2565 .field_2403 {
   box-shadow: 0px 1px 2px 0px gray;
   font-size: 2.5em;
   padding: 10px;
-  margin: 20px;
+  //margin: 20px;
```

Comment:
you can remove the margin line, but you will want to make sure you have a 10px margin on the small-button for the login pages so there is space between the big and small button:
![image](https://user-images.githubusercontent.com/67915167/118727732-db108f00-b7f8-11eb-8c6b-cc5276c5e77c.png)